Phoenix/hello/config/test.exs

```diff
@@ -7,7 +7,7 @@ import Config
 # Run `mix help test` for more information.
 config :hello, Hello.Repo,
   username: "postgres",
-  password: "postgres",
+  password: "5550142SaMy",
```

Comment:
@samu975 It is not a good practice to push secret variables to the repository. This is a security breach!